helm/providers/charts/cloudant/configMapFiles/myTask.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor
+# license agreements; and to You under the Apache License, Version 2.0.
+
+# installCatalog.sh expects the wsk cli to be $OPENWHISK_HOME/bin/wsk
+
+export OPENWHISK_HOME=/usr/local
+
+git clone https://github.com/apache/incubator-openwhisk-package-cloudant.git
+
+export DB_URL=http://$DB_USERNAME:$DB_PASSWORD@$DB_HOST:$DB_PORT
```

Comment:
Should also pass in DB_PROTOCOL (don't assume http).  See comment in installPkgCloudantJob.yaml.

Reply:
Done.